python/ray/autoscaler/_private/fake_docker/defaults.yaml

```diff
@@ -0,0 +1,47 @@
+# Example command to start a cluster with this config:
```

Comment:
will be getting rid of this file if not offering `ray up` functionality after all